.github/CODEOWNERS

```diff
@@ -11,6 +11,9 @@
 /packages/test-framework @bigcommerce/team-checkout
 /packages/workspace-tools @bigcommerce/team-checkout
 
+## Shared
+package.json @bigcommerce/team-checkout @bigcommerce/team-integrations
+package-lock.json @bigcommerce/team-checkout @bigcommerce/team-integrations
```

Comment:
May I ask you to add `@bigcommerce/team-paypal` here as well?

Reply:
Done 👍